iina/AppDelegate.swift

```diff
@@ -251,9 +251,6 @@ class AppDelegate: NSObject, NSApplicationDelegate {
   func applicationShouldTerminate(_ sender: NSApplication) -> NSApplication.TerminateReply {
     guard !PlayerCore.active.mainWindow.isWindowHidden else { return .terminateCancel }
```

Comment:
@saagarjha @LEoREo2247 Irrelevant about this pr, but why we cancel the termination when the main window is hidden due to pip?

Reply:
Hiding the window seemed to be treated same as closing them, which would cause the app to quit unexpectedly if the user is enabling the preference to quit the app when all window is closed. From what I see now, this can be moved into `applicationShouldTerminateAfterLastWindowClosed()` perhaps? or there may be a better way.

Reply:
Yeah, that sounds better. I've already moved that line.